bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
+To enter a command into the shell, type it in where the blinking cursor appears and press `Enter` or `return` on your keyboard.
 
-### Knowledge Check 1
+### Navigation Commands
 
-What is the name of the pre-installed program used to run shell scripts on a Mac?
+Just like when you navigate any file system, it is important to know where you are, see what files and folders are in that location, and move to other locations.
 
-[( )] Together
-[( )] MSL (Mac Subsystem for Linux)
-[(X)] Terminal
-[( )] Ubuntu
-***********************************************************************
-<div class = "answer">
+**Where am I?**
 
-WSL is the Windows utility for running shell. Ubuntu is a well known Linux operating system that is often used as the default shell scripting environment.
+The command `pwd`, which stands for **p**resent **w**orking **d**irectory, will return the path through your directory system to your current location.
 
-</div>
-***********************************************************************
+**What is here?**
 
-## Setting up a User/Home Directory
+The bash shell doesn't show the icons of the files and directories in your present working directory. Instead you will need to ask for a list of the directory's contents with the command `ls`.
 
-- As you get comfortable in bash, it is important to create a default home directory in an easily accessible place.
-- It is recommended that you use the native user directory in the home directory on your computer. Your user directory contains a folder named “bin” which is the root directory of any unix like operating system.
-- Bin is short for “binary” and is where you can store any configuration or executable files for programs you run on the CLI. This will be a good place to store bash scripts (.sh files) once you have become comfortable executing commands and learning syntax.
+**How do I move around?**
 
-![Accessing the user/home directory on a Mac.](media/user_directory.png)
+The command `cd` lets you **c**hange **d**irectory. Unlike the previous two commands, `cd` requires you to provide additional information: the directory you want to move to. You can move to a location relative to your current location, or you can move anywhere directly if you know the path to get there.
 
-## Useful Commands and Syntax
+- Move to a directory relative to your present working directory:
 
-In our installation section, we covered how accessing CLI tools to run bash scripts is quite uncomplicated on either operating system. Similarly, bash commands and syntax are approachable and integrate well to allow for a gradual increasing in the complexity of utilities run.
+  * If the directory `directory_name` is in your present working directory (i.e. it is one of the listed items when you enter `ls`) you can navigate into it with the command `cd directory_name`
 
-### Commands
+  * To move to the parent directory of your current location, use `cd ..`
```

Comment:
```suggestion
  * To navigate to the parent directory of your current location, use `cd ..`
```
"move" is ambiguous because we "move" files also.  Just a thought!